Contributing Instructions (#21)
InnerSourceCommons/ispo-working-group: CONTRIBUTING.md

Welcome!
Every person creating InnerSource-related content or tools within their company should contribute to this working group.
This working group helps all to get feedback on this work and to share it so that others can leverage it as well.
Check out our [README.md] for more details.

## 3 Ways to Contribute

1. Contribute your InnerSource-related content.

    * We track contributions on this [Kanban board].
    * Get access to contribute to the board by [joining the ISPO Working Group team].

>     * Once you have access, create a new card in the **No Status** column describing the content that you want to contribute and what challenge it helps to solve.

Reviewer: Easier to read with:

```suggestion
Create a new card in the **No Status** column describing your contribution and the challenge it helps solve.
```

Do we need to add _Once you have access_ or the content, as both are clearly implied?

    * Assign the new card to yourself.

>     * Join [#ispo-working-group] channel in the [InnerSource Commons Slack] to discover when is the next working group meeting.

Reviewer: Recommend:

```suggestion
    * Learn more about the next Working Group meeting by joining [#ispo-working-group] in [InnerSource Commons Slack].
```

>     * In the meeting, discuss your card and what you are planning to contribute and agree on next steps.

Reviewer: Recommend:

```suggestion
    * During the meeting, discuss your card and what you plan to contribute.
```

2. Review others' content.

>     * Open the [Kanban board] and find something that is interesting to you to review.

Reviewer: Recommend:

```suggestion
    * Find something interesting on the [Kanban board].
```

>     * Leave a comment on the card indicating that you are interested in reviewing.

Reviewer: Recommend:

```suggestion
    * Write a comment on the card indicating your interest.
```

3. Attend working group meeting.

>     * Join [#ispo-working-group] channel in the [InnerSource Commons Slack] to discover when is the next working group meeting.

Reviewer: Recommend:

Shorter, and Community is an invitational word.

```suggestion
    * The [InnerSource Commons Slack] [#ispo-working-group] community will post the next meeting.
```

## Project Board

> This working group tracks the discovery, production, and spread of solutions to challenges with InnerSource.

Reviewer: Recommend:

```suggestion
 InnerSource solutions to challenges are discovered, produced, and spread in this working group.
```

We track this production on a [Kanban board] with the following columns:

1. **Challenge**.  This means that we've identified such-and-such a thing as a challenge.
1. **Proposed**.  We have a proposal for a solution to the challenge.
This could be to write an article, code up a tool, create a training video, etc.
1. **Internal**.  We have implemented the solution internally in a company.
1. **External**.  We have made the solution available externally.
1. **Validated**.  More than one company is using the solution.

> We have a regular meeting where we will list new challenges, discuss proposed solutions, and demo finished solutions.

Reviewer: * Less wordy
* Finished is implied by _solution_

```suggestion
A regular meeting proposes new challenges, discusses ideas, and demos solutions.
```

> It is expected that the implementation of many solutions may be contributions to other working groups in the InnerSource Commons (e.g. Patterns, Learning Path).

Reviewer 2: > In many cases, solutions are shareable with other InnerSource Commons working groups; (e.g. [Patterns](https://github.com/InnerSourceCommons/InnerSourcePatterns), [Learning Path](https://github.com/InnerSourceCommons/InnerSourceLearningPath)).

I like your proposal @jeffabailey! Would change it a little bit to make it less formal and to use the "working groups" term that we have already established. Also added the links as you suggested.

> It is expected that the implementation of many solutions may be contributions to other working groups in the InnerSource Commons (e.g. Patterns, Learning Path).

Reviewer 2: ```suggestion
In many cases, solutions are shareable with other InnerSource Commons working groups; (e.g. [Patterns](https://github.com/InnerSourceCommons/InnerSourcePatterns), [Learning Path](https://github.com/InnerSourceCommons/InnerSourceLearningPath)).
```

I like your proposal @jeffabailey! Would change it a little bit to make it less formal and to use the "working groups" term that we have already established. Also added the links as you suggested.
